plugins-fixed/PingPongPan.lv2/PingPongPan_dsp.ttl

```diff
@@ -52,6 +52,8 @@
         lv2:default 50.000000 ;
         lv2:minimum 0.000000 ;
         lv2:maximum 100.000000 ;
+        units:unit units:hz ;
```

Comment:
I made this plugin, this is certainly not Hz!
This value is percentage, without any real meaning on what it means (just goes from full speed to full stop)